uaclient/status.py

```diff
@@ -132,10 +129,14 @@ def format_tabular(status):
         return MESSAGE_UNATTACHED
     tech_support_level = status['techSupportLevel']
 
-    content = [STATUS_HEADER_TMPL.format(
-        account=status['account'], subscription=status['subscription'])]
-    if status['origin'] != 'free':  # All free accounts will have 'free' origin
-        content.append('Valid until: ' + str(status['expires']))
+    content = []
+    if status['origin'] == 'free':
+        content.append('     Account: ' + status['account'])
```

Comment:
I think this would be preferable using the spacing that Python format templates provide (so that we can easily change the spacing in future if necessary). See `STATUS_TMPL` (in context of this diff) for a pointer towards how to do that.

Reply:
I pushed a change and I am looking for thoughts on how to make that cleaner. It looks messy and you can tell I haven't dealt with fstrings very often.